test/argent_account.py

```diff
@@ -280,12 +376,67 @@ async def test_escape_signer(get_starknet, account_factory):
 
     # should escape after the security period
     assert (await account.get_signer().call()).result.signer == (signer.public_key)
-    await sender.send_transaction(account.contract_address, 'escape_signer', [new_signer.public_key], [guardian])
+    tx_exec_info = await sender.send_transaction(account.contract_address, 'escape_signer', [new_signer.public_key], [guardian])
+
+    assert_event_emmited(
+        tx_exec_info,
+        from_address=account.contract_address,
+        name='signer_escaped',
+        data=[new_signer.public_key]
+    )
+
     assert (await account.get_signer().call()).result.signer == (new_signer.public_key)
 
     # escape should be cleared
     escape = (await account.get_escape().call()).result
-    assert (escape.active_at == 0 and escape.caller_is_signer == 0)
+    assert (escape.active_at == 0 and escape.type == 0)
+
+@pytest.mark.asyncio
+async def test_trigger_escape_signer_by_guardian_overridden(get_starknet, account_factory):
```

Comment:
I find the name of the test confusing :-)

Maybe rename with `test_signer_overrides_trigger_escape_signer`?